edlib/include/edlib.h

```diff
@@ -39,7 +29,7 @@ extern "C" {
     /**
      * Alignment methods - how should Edlib treat gaps before and after query?
      */
-    EDLIB_API typedef enum {
+    typedef enum {
```

Comment:
Probably stupid question, but why don't typedefs / enums / structs also need EDLIB_API in front? Compiler threats those differently somehow, doesn't care about them during linking?

Reply:
Because typedef is a definition which is part of header file, there is no symbol to export in shared lib.